src/Rollbar.elm

```diff
@@ -164,19 +167,19 @@ levelToString report =
             "warning"
 
 
-sendWithTime : Token -> Scope -> Environment -> Int -> Level -> String -> Dict String Value -> Time -> Task Http.Error Uuid
-sendWithTime token scope environment maxRetryAttempts level message metadata time =
+sendWithTime : Token -> Scope -> Environment -> Int -> Level -> String -> Dict String Value -> Posix -> Task Http.Error Uuid
```

Comment:
Whoops, this should actually be a `Float` and not a `Time.Posix`! 😅 

`Time.Posix` specifically refers to a UTC timestamp - a particular point in time - whereas this is a duration (in milliseconds).

Reply:
I am confused by this.  `sendWIthTime` is called by the `Task` created by `Time.now` in `send`. If this is incorrect, some guidance would be appreciated.

Reply:
You're totally right, my mistake! 😅